lib/providers/background_sounds/background_sounds_provider.dart

```diff
@@ -7,10 +7,10 @@ import 'package:Medito/repositories/repositories.dart';
 import 'package:Medito/services/shared_preference/shared_preferences_service.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:riverpod_annotation/riverpod_annotation.dart';
-part 'background_sounds_viewmodel.g.dart';
+part 'background_sounds_provider.g.dart';
 
 @riverpod
-Future<List<BackgroundSoundsModel>> backgroundSounds(BackgroundSoundsRef ref) {
```

Comment:
what's the rationale behind removing these types? just wondering

Reply:
It was showing me a warning with a yellow line to remove the type so, that's why I removed the type.

Reply:
Strange I would have thought it would prefer the type! Oh well

Reply:
It's happening in some of the places mostly with the providers.